wp-content/plugins/wporg-learn/views/metabox-meeting-language.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * Template for Meeting Language metabox
+ */
+
+/** @var WP_Post $post */
+/** @var array $locales */
+/** @var string $language */
+?>
+
+<?php wp_nonce_field( 'meeting-metaboxes', 'meeting-metabox-nonce' ); ?>
+
+<p>
+	<select id="meeting-language" name="meeting-language" style="width: 100%;">
```

Comment:
A11y issue with this id, it ends up being a duplicate:
![Screen Shot 2022-12-05 at 3 14 03 PM](https://user-images.githubusercontent.com/1017872/205534531-17fe36e6-166c-42b5-9bf7-1f781ca1b85b.jpg)

Suggest changing this to:
```suggestion
	<select id="meeting-language-selector" name="meeting-language" aria-label="Meeting Language" style="width: 100%;">
```

Aria label added as we don't want to have a visible label, it will just be a duplicate of the meta box title.